source/features/update-pr-from-base-branch.tsx

```diff
@@ -10,11 +10,30 @@ import features from '../feature-manager';
 import observe from '../helpers/selector-observer';
 import * as api from '../github-helpers/api';
 import {getBranches} from '../github-helpers/pr-branches';
-import getPrInfo from '../github-helpers/get-pr-info';
-import {getConversationNumber} from '../github-helpers';
+import getPrInfo, {PullRequestInfo} from '../github-helpers/get-pr-info';
 import showToast from '../github-helpers/toast';
+import pluralize from '../helpers/pluralize';
+import {buildRepoURL, getConversationNumber} from '../github-helpers';
 import createMergeabilityRow from '../github-widgets/mergeability-row';
 import selectHas from '../helpers/select-has';
+import {linkifyCommit} from '../github-helpers/dom-formatters';
+import { removeTextNodeContaining } from '../helpers/dom-utils';
+
+function getBaseCommitNotice(prInfo: PullRequestInfo): JSX.Element {
+	const {head} = getBranches();
+	const commit = linkifyCommit(prInfo.baseRefOid);
+	const count = pluralize(prInfo.behindBy, '$$ commit', '$$ commits');
+	const countLink = (
+		// Adjust for cross-repo URLs like
+		// https://github.com/refined-github/sandbox/compare/default-a...bfred-it-org:github-sandbox:branch/for-pr
+		<a href={buildRepoURL('compare', `${prInfo.baseRefOid}...${head.full.replace(':', '')}`)}>
+			{count}
+		</a>
+	);
+	return (
+		<>This branch is {countLink} behind the base branch (base commit: {commit})</>
```

Comment:
Ideas? This kinda repeats the title, we can shorten it further

Reply:
Interestingly the count seems wrong, there's currently just one commit.

PR: https://github.com/refined-github/sandbox/pull/60

Base commit: https://github.com/refined-github/sandbox/commits/2723dc763d512e421d2562c1da04b6cb14ba8a12

Base branch: https://github.com/refined-github/sandbox/commits/default-a

Comparison: https://github.com/refined-github/sandbox/compare/2723dc76...default-a

You can see commit `2723` (base commit) is the second one and only `59a2` is missing.

Reply:
This could work:

<img width="434" alt="Screenshot 19" src="https://user-images.githubusercontent.com/1402241/232424604-0f6da872-cfea-42b8-b4cb-d6ee5892ac52.png">

Ideas on how to integrate the "base commit" info? It's not a super clear concept 🤔

Reply:
The drawback of this position is that the DOM may vary a lot, which is actually why we pulled `update-pr-from-base-branch` out of the mergeability box years ago.

It's easier for us to handle it now, but there's still some UI variability that makes this a little complex 🥲 (e.g. PR with conflicts don't show `pr-base-commit` right now)